docs/features/if_statements.md

```diff
@@ -0,0 +1,32 @@
+# If Statements
+
+```js
+// Simple if statement that will always be false
+
+if (1 /= 1) { // usually if (1 != 1) in other langugaes
```

Comment:
Maybe it's more idiomatic to use `not 1 = 1`?

Reply:
This seems like it is less useful than getting the `/=` syntax out of the way.

Reply:
perhaps